build_rules.gradle

```diff
@@ -1253,6 +1254,125 @@ ext.applyGrpcNature = {
 
 /*************************************************************************************************/
 
+ext.applyPortabilityNature = {
+  println "applyPortabilityNature with " + (it ? "$it" : "default configuration") + " for project $project.name"
+  applyJavaNature(enableFindbugs: false, enableErrorProne: false, shadowClosure: {
```

Comment:
Will this shading configuration eventually be common to all java projects (not just portability)?

Reply:
Eventually all projects will not do any shading and instead use normal dependencies (no more `shadow`, `shadowTest`). The only reason why I place this into a function right now is that I want to repeat this same logic across `model/pipeline`, `model/job-management`, `model/fn-execution`.